packages/css/src/components/heading/README.md

```diff
@@ -2,15 +2,14 @@
 
 # Heading
 
-A heading conveys information about the content below it.
-Use headings to allow the user to grasp the structure of the page quickly.
+Communicates the organization of the content on a page.
```

Comment:
I don't think this is an improvement, it's a bit vague. Why do you want to change it?

Reply:
I borrowed it from the [W3C WAI](https://www.w3.org/WAI/tutorials/page-structure/headings/). I think the rationale should not include usage information and should not start with the component name. I intend to update the rationale for all components like this and thought I could take this one along already.

I’ve restored the “A heading describes the content beneath it.” so we won’t miss that as well?

Reply:
It's still really vague imo. We could just do this, and remove the first guideline:

```suggestion
Describes the content that follows it.
Use Headings to allow the user to grasp the structure of the page quickly.
```